### ISSUE TITLE
Intention of Converting Entities in RadGraph into Non-sensitive Numbers

### ISSUE THREAD

**Issue by YatingPan** [xiongyiheng/Prior-RadGraphFormer#1]
Hi, I have a question about the code in preprocessing of RadGraph. What's the intention of converting all entities' names into non-sensitive numbers? I am also working on the same dataset and wondering if there are some special concerns.

Thank you and look forward to hearing from you!

**Comment by xiongyiheng** [comment]
Hi Yating,
It simply means making text labels to number for training i.e. lung -> 0, plural -> 1, left -> 2. No more special concerns :)

**Comment by YatingPan** [comment]
Thanks for quick answer! Is there any advantages to do this instead of directly feeding the text labels to training? To save training time or memory?

**Comment by xiongyiheng** [comment]
Hi,
our loss function is essentially cross entropy loss for classification, so we must feed numbers to represent class labels for it insead of texts

**Comment by YatingPan** [comment]
Thank you for answering! Now my confusion is solved. Thanks a lot!
